### PR TITLE
MockGen header comment: trim .exe suffix on Windows

### PR DESCRIPTION
**Pull request by craig65535**
To address #118

### CODE REVIEW

File: mockgen/mockgen.go [hunk 1]
```diff
@@ -316,7 +317,11 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 	g.p("// Generated by this command:")
 	g.p("//")
 	// only log the name of the executable, not the full path
-	g.p("//\t%v", strings.Join(append([]string{filepath.Base(os.Args[0])}, os.Args[1:]...), " "))
+	name := filepath.Base(os.Args[0])
+	if runtime.GOOS == `windows` && strings.HasSuffix(name, `.exe`) {
+		name = strings.TrimSuffix(name, `.exe`)
```
Comment (marten-seemann) [inline mockgen/mockgen.go:322]: Why not use normal quotation marks?

Comment (craig65535) [reply mockgen/mockgen.go:322]: It's a habit from the codebase I work on day-to-day. I'll change it.

File: mockgen/mockgen.go [hunk 2]
```diff
@@ -316,7 +317,11 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 	g.p("// Generated by this command:")
 	g.p("//")
 	// only log the name of the executable, not the full path
-	g.p("//\t%v", strings.Join(append([]string{filepath.Base(os.Args[0])}, os.Args[1:]...), " "))
+	name := filepath.Base(os.Args[0])
+	if runtime.GOOS == "windows" && strings.HasSuffix(name, ".exe") {
```
Comment (marten-seemann) [inline mockgen/mockgen.go:321]:
```suggestion
	if runtime.GOOS == "windows" {
```

**Review by marten-seemann — APPROVED** [review]

**Review by marten-seemann — APPROVED** [review]

**Comment by craig65535** [comment]
@r-hang Are you able to start the workflow on this PR? Thank you!

**Comment by craig65535** [comment]
@tchung1118 Are you able to start the workflow on this PR? Thank you!

**Review by r-hang — APPROVED** [review]
Sorry for the delay, lgtm.

**Pull request merged**